crates/ruff_cli/Cargo.toml

```diff
@@ -60,6 +61,9 @@ walkdir = { version = "2.3.2" }
 assert_cmd = { version = "2.0.8" }
 ureq = { version = "2.6.2", features = [] }
 
+[features]
+jupyter_notebook = ["ruff/jupyter_notebook"]
```

Comment:
(We need to do this for `logical_lines` too I think, not part of this PR, just a note to self...)